client/my-sites/email/email-management/titan-control-panel-redirect/index.jsx

```diff
@@ -90,7 +90,7 @@ class TitanControlPanelRedirect extends React.Component {
 				<EmptyContent illustration="" title="">
 					<Card>
 						<Spinner size={ 40 } />
-						<h1>{ translate( 'Redirecting you to your Email Control Panel' ) }</h1>
+						<h1>{ translate( 'Redirecting you to your Professional Email Control Panel' ) }</h1>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.